gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_0_0/schema.yml

```diff
@@ -15,10 +15,11 @@ databaseChangeLog:
               - column: {name: parent_id, type: nvarchar(64), constraints: { nullable: true } }
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.